features/consent/use-allow-group.mutation.ts

```diff
@@ -126,6 +131,10 @@ export const getAllowGroupMutationOptions = (
           state: context.previousConsent,
         },
       });
+      addConversationToUnknownConsentConversationsQuery({
+        account,
+        conversation: group,
```

Comment:
do we want to be consistent and use topic (id) instead of object (group)

Reply:
you have to pass the whole object here because it's not even existing in cache yet